packages/@react-aria/searchfield/src/useSearchField.ts

```diff
@@ -79,6 +79,10 @@ export function useSearchField(
     }
   };
 
+  let onPressStart = () => {
+    inputRef.current.focus();
```

Comment:
Does getting rid of the focus call in the click handler work?

Reply:
I don't follow? you mean in this one?
```
  let onClearButtonClick = () => {
    state.setValue('');
    inputRef.current.focus();

    if (onClear) {
      onClear();
    }
  };
```

We could probably drop that one now, keeping this change as well